packages/panzer/dof-mgr/src/Panzer_GlobalIndexer.hpp

```diff
@@ -255,8 +255,8 @@ class GlobalIndexer {
        functor.num_dofs = lids.extent(1);
 
 #ifdef PANZER_DEBUG
-     TEUCHOS_ASSERT(functor.local_lids.extent(1) >= num_dofs);
-     TEUCHOS_ASSERT(functor.global_lids.extent(1) >= num_dofs);
+     TEUCHOS_ASSERT(functor.local_lids.extent_int(1) >= num_dofs);
+     TEUCHOS_ASSERT(functor.global_lids.extent_int(1) >= num_dofs);
```

Comment:
I would avoid using the extent_int() functions in kokkos views. I think they plan to deprecate them eventually. Maybe just use a cast instead?